### ISSUE TITLE
Inconsistent indentation of Notes

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag#4436]
In  [5.3.1 Required Components of a Conformance Claim](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#conformance-required), Note 3 has a different indentation than Note 1 and Note 2.  Based on how Notes are treated in the previous section, it _might_ be the case that the formatting of Note 1 and Note 2 are the ones needing correction.  It might also be the case that all three notes are indented correctly, per the style guide.  Regardless, in context of that section, it looks very odd.  WCAG 2.1 has the same [formatting of this section](https://www.w3.org/TR/2025/REC-WCAG21-20250506/#conformance-required), and the items flagged below are also applicable to 2.1.

[Note 3 for abbreviation](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#dfn-abbreviations) in the glossary I _think_ is indented correctly, but it looks odd in context.

The [Keyboard Interface definition](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#dfn-keyboard-interface) is the only instance of an Example within a Note.  This seems like a mistake as other places in glossary intermix Notes and Examples.

In the glossary, _most_ example boxes have consistent formatting.  These do not use the word "example" _except_ for the leading label for the box.  It's very nice!  A minority of the  glossary terms use both the formatting and the label, but then include phrases like "for example" or "examples are".  These outliers are a little bit confusing (but _only_ a little bit) and should probably be made consistent with the others.

**Comment by patrickhlauke** [comment]
> In [5.3.1 Required Components of a Conformance Claim](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#conformance-required), Note 3 has a different indentation than Note 1 and Note 2.

notes 1 and 2 refer directly / come under bullet point 4 specifically. note 3 applies to everything in 5.3.1 itself and are outside of the list. the indentation reflects this.

> [Note 3 for abbreviation](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#dfn-abbreviations) in the glossary I think is indented correctly, but it looks odd in context.

it's indeed correct. note 3 applies to everything under "abbreviation" just like note 1. note 2 only relates to the bullet point 1

> The [Keyboard Interface definition](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#dfn-keyboard-interface) is the only instance of an Example within a Note. This seems like a mistake as other places in glossary intermix Notes and Examples.

the example is specifically for what's in note 1

in short, these all look correct due to how they're scoped.

**Comment by bruce-usab** [comment]
> the example is specifically for what's in note 1

I will re-re-review, but I thought I found instances where examples were specific to the preceding note, but not embedded within the note.

> in short, these all look correct due to how they're scoped.

I can accept that this is the case.  Thanks for taking a look!

**Comment by mbgower** [comment]
I'm inclined to agree with Bruce that there's something a little daffy going on here. However, it's not isolated to the alignment.

 I would argue that Note 1 is not a note at all, but part of the definition,  as are the numbered items below on initialisms and acronyms. 

Note 2 should become Note 1 and be adjusted to read something like "Initialisms are not defined in all languages."

Personally, I find it _really_ problematic when notes are numbered through different hierarchies of document structure. I would also be inclined to remove the numbering in this case and leave each of the remaining note as unnumbered notes in their respective conten.

I'm going to create a PR to show how I think this should be. Note that this will then be a proposed normative change.

**Comment by alastc** [comment]
Just to add for one of the examples, the notes under the abbreviation defintion has been mangled by the transitions between formats.

Under [WCAG 2.0](https://www.w3.org/TR/WCAG20/#abbreviationsdef), the first note originally wrapped all the content until the last note. 

Under WCAG 2.1 onwards (e.g. in [WCAG 2.2](https://www.w3.org/TR/WCAG22/#dfn-abbreviations)), it has been mangled so the Note 1 doesn't wrap the rest of the content.

I suggest updating that to match the original. No nested note formatting, update the nested note to just be a plain text thing.

**Comment by mbgower** [comment]
I believe we need to mark this as errata. Although we have confirmed that this is being returned to match the original definition note, it _appears_ to be a normative change to a definition. As such, I've marked it as an erratum, but set it as non-normative to clarify.

**Comment by bruce-usab** [comment]
Discussed on backlog call 6/20.  Closed by #4452
